Resources/doc/reference/form_field_definition.rst

```diff
@@ -227,16 +227,23 @@ You can easily add a new ``galleryHasMedias`` row by defining one of these optio
                 ->add('enabled')
                 ->add('name')
                 ->add('defaultFormat')
-                ->add('galleryHasMedias', 'sonata_type_collection', array(), array(
-                    'edit' => 'inline',
-                    'inline' => 'table',
-                    'sortable' => 'position',
-                    'limit' => 3
+                ->add('galleryHasMedias', 'sonata_type_collection', array(
```

Comment:
Not sure why that would matter, but here it is : https://github.com/symfony/form/blob/master/CHANGELOG.md